### PR TITLE
Feature  bed 20

### PR DESCRIPTION
**Pull request by ame-test**
## **What changes does this PR bring?**

- Adds Email address to the form.
- Makes it persist via cookies.
- Added it to the table when checking the answers for a gar.
- Is available on the PDF viewer.
- will turn to a PDF with that field.

Attached are screenshots of the change.

The tests have been updated with it --> though this changes:
- Has been not confirmed by the tests, since the tests are not passing yet.
- No obvious refactors made --> as tests are not passing

### Adds Email address to the form.
Screenshot 2023-04-14 at 10 36 40
### Makes it persist via cookies.
Screenshot 2023-04-14 at 10 37 09
### Added it to the table when checking the answers for a gar.
Screenshot 2023-04-14 at 10 35 49
### Is available on the PDF viewer.
Screenshot 2023-04-14 at 10 37 25
### Will turn to a PDF with that field.
.Screenshot 2023-04-14 at 10 37 32

### CODE REVIEW

File: src/app/garfile/responsibleperson/index.njk [hunk 1]
```diff
@@ -58,6 +58,20 @@
               aria-describedby="responsibleContactNo-hint{{ ' responsibleContactNo-error' if errors | containsError('responsibleContactNo') }}">
           </div>
 
+          <div class="govuk-form-group govuk-!-margin-bottom-6">
```
Comment (anushasivakumar) [inline src/app/garfile/responsibleperson/index.njk:61]: Have you tested the error validation for this field? Were the errors displayed properly? Can you please check that as one of the macros to add a css class for error seems to be missing here. It is present in a few other fields in this file.  
`{{ m.form_group_class('responsibleContactNo', errors) }}`

**Review by anushasivakumar — CHANGES_REQUESTED** [review]
Good job overall, added some minor comments. Thanks!

File: src/app/garfile/responsibleperson/get.controller.js [hunk 2]
```diff
@@ -18,10 +18,13 @@ module.exports = (req, res) => {
     .then((apiResponse) => {
       const gar = JSON.parse(apiResponse);
 
+      logger.debug(`gar : ${JSON.stringify(gar)}`);
+
```
[the remark this answers is not in the document]
Comment (ame-test) [reply src/app/garfile/responsibleperson/get.controller.js]: Agreed, the logs were added for development purposes.

I have removed them.

File: src/app/garfile/responsibleperson/index.njk (hunk 1 above)
Comment (ame-test) [reply src/app/garfile/responsibleperson/index.njk:61]: Absolutely, I should have provided the error messages as part of the screenshots.

**Below is the error messages for:**

The top page if the email address is missing 
Screenshot 2023-04-14 at 14 03 46

The error nearer the input field.
Screenshot 2023-04-14 at 14 03 54

**Review by anushasivakumar — APPROVED** [review]

**Pull request merged**